core/src/dispatch/mod.rs

```diff
@@ -70,7 +77,7 @@ pub struct NetworkConfig {
 }
 
 impl NetworkConfig {
-    /// Create a new config with `addr` and protocol [TCP](Transport::Tcp)
+    /// Create a new config with `addr` and protocol [Tcp](Transport::Tcp)
```

Comment:
I kinda left all of these on purpose upper case. When talking about the protocol itself the upper case acronym is correct. Only when referring to it as a type do Rust conventions insist that we do the funky lower casing. But if you read "Tcp" by itself in doc text it looks really weird, so it makes sense to leave it TCP in the link text.